src/main/java/uk/gov/ons/ctp/response/action/scheduled/distribution/ActionDistributor.java

```diff
@@ -35,26 +27,17 @@
       Sets.immutableEnumSet(ActionState.SUBMITTED, ActionState.CANCEL_SUBMITTED);
 
   private ActionRepository actionRepo;
-  private ActionCaseRepository actionCaseRepo;
   private ActionTypeRepository actionTypeRepo;
 
   private ActionProcessingService businessActionProcessingService;
 
-  private StateTransitionManager<ActionState, ActionDTO.ActionEvent>
-      actionSvcStateTransitionManager;
-
   public ActionDistributor(
-      AppConfig appConfig,
       ActionRepository actionRepo,
-      ActionCaseRepository actionCaseRepo,
       ActionTypeRepository actionTypeRepo,
-      @Qualifier("business") ActionProcessingService businessActionProcessingService,
-      StateTransitionManager<ActionState, ActionDTO.ActionEvent> actionSvcStateTransitionManager) {
+      ActionProcessingService businessActionProcessingService) {
```

Comment:
Yes I got rid of the DefaultBusinessActionProcessing service because it was a horrible unnecessary composition thing. The SocialActionProcessing service was deleted in a previous change set, so now there's only one. 

I'll rename 👍